sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/TaskResourceMapper.java

```diff
@@ -51,14 +52,15 @@ public String toString() {
         }
     }
 
-    public TaskResourceMapper(TaskSpec taskSpec, Protos.TaskInfo taskInfo) {
+    public TaskResourceMapper(TaskSpec taskSpec, Protos.TaskInfo taskInfo, boolean useDefaultExecutor) {
         this.taskSpecName = taskSpec.getName();
         this.resourceSpecs = new ArrayList<>();
         this.resourceSpecs.addAll(taskSpec.getResourceSet().getResources());
         this.resourceSpecs.addAll(taskSpec.getResourceSet().getVolumes());
         this.resources = taskInfo.getResourcesList();
         this.taskEnv = EnvUtils.toMap(taskInfo.getCommand().getEnvironment());
         this.evaluationStages = getEvaluationStagesInternal();
+        this.useDefaultExecutor = useDefaultExecutor;
```

Comment:
Subjective: may as well just pass it into the static functions below. For example taskSpecName is already being passed in despite also being a member.